### PR TITLE
CID-2466-Documentation for List Users with Role API call

### PR DESCRIPTION
**Pull request by abhatt1250**
- Create documentation for List Users with Role API call
- correcting api-docs/api-reference/methods/users/post-user-information-and-password-v2.0.rst

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

File: api-docs/api-reference/methods/roles/get-list-users-with-role-v2.0.rst [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+    GET /v2.0/OS-KSADM/roles/{roleId}/RAX-AUTH/users
+
+This operation returns a list of users with detailed account information about
+each user, including email, name, user ID, phone pin state, account
+configuration, and status information assigned to the specified role ID.
+
+.. note::
+
+  - Users with the ``identity:service-admin`` role can list users with the
```
Comment (premaseem) [inline api-docs/api-reference/methods/roles/get-list-users-with-role-v2.0.rst:16]: usually, we do not expose ``identity:service-admin``  info in public-facing docs. @kidster can confirm if we need it here or not.

Comment (kidster) [reply api-docs/api-reference/methods/roles/get-list-users-with-role-v2.0.rst:16]: Yes, that is correct. We need to remove the sections of `identity:service-admin` and `identity:admin`. Public docs should only show information relevant to a customer.

File: api-docs/api-reference/methods/roles/get-list-users-with-role-v2.0.rst [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    ``identity:admin`` role, ``identity:user-admin`` role and sub-users assigned
+    to a specified role.
+
+  - Users with the ``identity:admin`` role can list users with the
```
Comment (kidster) [inline api-docs/api-reference/methods/roles/get-list-users-with-role-v2.0.rst:20]: Remove.

**Pull request merged**